meta-resin-common/recipes-connectivity/networkmanager/networkmanager_1.6.2.bb

```diff
@@ -18,15 +18,16 @@ inherit gnomebase gettext systemd bluetooth vala
 SRC_URI = " \
     ${GNOME_MIRROR}/NetworkManager/${@gnome_verdir("${PV}")}/NetworkManager-${PV}.tar.xz \
 "
-SRC_URI[md5sum] = "59b5b5585467756787c6cb535cbd306e"
-SRC_URI[sha256sum] = "a864e347ddf6da8dabd40e0185b8c10a655d4a94b45cbaa2b3bb4b5e8360d204"
+SRC_URI[md5sum] = "89c975afe19fbac854191edb6e9bcd3b"
+SRC_URI[sha256sum] = "b22b6f55cde37bec4982f9be4b1808a21101d807a05049c670116d95059a26f9"
 
 S = "${WORKDIR}/NetworkManager-${PV}"
 
 EXTRA_OECONF = " \
     --disable-ifcfg-rh \
     --disable-ifnet \
     --disable-ifcfg-suse \
+    --disable-json-validation \
```

Comment:
--disable-json-validation: This parameter allows building without Jansson, which is needed for “team” configuration validation (team is one way of bonding network interfaces for increased throughput).

Sounds reasonable